network-mux/network-mux.cabal

```diff
@@ -161,3 +161,36 @@ executable mux-demo
                        -Wpartial-fields
                        -Widentities
                        -Wredundant-constraints
+
+executable cardano-ping
+  hs-source-dirs:      demo, test
+  main-is:             cardano-ping.hs
+  other-modules:       Test.Mux.ReqResp
+  build-depends:       base,
+                       directory,
+                       network-mux,
+                       io-sim-classes,
+                       contra-tracer,
+                       stm,
+
+                       binary,
+                       bytestring,
+                       cborg,
+                       network,
+                       serialise,
+                       tdigest,
+                       text
+  if os(windows)
+    buildable:         False
+  else
+    buildable:         True
+  default-language:    Haskell2010
+  ghc-options:         -threaded
```

Comment:
Maybe we can use the [common](https://cabal.readthedocs.io/en/3.4/cabal-package.html#pkg-section-common-common) stanza for common `ghc` options?  It requires`cabal-version: 2.2`.

Reply:
cabal-version 2.2 shouldn't be a problem, we already require a far newer version.